modules/rosa-sdpolicy-storage.adoc

```diff
@@ -27,4 +27,4 @@ Each cloud provider has its own limits for how many PVs can be attached to a sin
 
 == Shared Storage (RWX)
 
-The AWS CSI Driver can be used to provide RWX support for {product-title}. A community Operator is provided to simplify setup.
+The link:https://access.redhat.com/articles/5025181[AWS CSI Driver] can be used to provide RWX support for {product-title}. A community Operator is provided to simplify setup.
```

Comment:
Instead of hyperlinking the words "AWS CSI Driver" (which makes me think that it is a link to the CSI driver), can we add a sentence at the end to state something like "please see the following article for further information."

Reply:
Thank you, @0kashi . Your feedback is incorporated.